src/pages/en/guides/markdown-content.md

```diff
@@ -178,113 +78,107 @@ For Markdown files, the `content` prop also has an `astro` property which holds
 }
 ```
 
-> Keep in mind that the only guaranteed properties coming from the `content` prop are `astro` and `url`.
+> 💡 `astro` and `url` are the only guaranteed properties provided by Astro in the `content` prop. The rest of the object is defined by your front matter variables.
 
-### Images and videos
+### Variables 
+
+Front matter variables can be used directly in your Markdown as properties of the `frontmatter` object.
+
+```markdown
+---
+layout: ../layouts/BaseLayout.astro
+author: Leon
+age: 42
+---
+# About the Author
 
-Using images or videos follows Astro’s normal import rules:
+{frontmatter.author} is {frontmatter.age} and lives in Toronto, Canada.
 
-- Place them in the `public/` as explained on the [project-structure page](/en/core-concepts/project-structure/#public)
-  - Example: Image is located at `/public/assets/img/astonaut.png` → Markdown: `![Astronaut](/assets/img/astronaut.png)`
-- Or use `import` as explained on the [imports page](/en/guides/imports#other-assets) (when using Astro’s Markdown Component)
+```
 
 ### Components
 
-You can import components from any framework into your markdown file and use it along the markdown syntax. Front matter variables are available to your components from the `frontmatter` object.
+You can import components into your Markdown file with `setup` and use them along with Markdown syntax. The `frontmatter` object is also available to any imported components.
 
 ```markdown
 ---
 layout: ../layouts/BaseLayout.astro
 setup: | 
   import Author from '../../components/Author.astro'
-  import MyReactComponent from '../components/MyReactComponent.jsx'
+  import Biography from '../components/Biography.jsx'
 author: Leon
 ---
-# Hydrating on Visibility
-
-<MyReactComponent client:visible >
-# Hello world!
-</MyReactComponent>
+# About the Author
 
 <Author name={frontmatter.author}/>
-```
 
+<Biography client:visible>
+{frontmatter.author} lives in Toronto, Canada and enjoys photography.
+</Biography>
 
-### Markdown draft pages
+```
 
-By default, Astro excludes `draft` Markdown pages when building your site. This is a built-in, Astro solution for marking individual pages or posts as "unpublished" and excluding them from the site build. No page will be created on your site for a Markdown page that includes `draft: true` in its front matter.
+### Draft Pages
 
-To enable building of draft pages, you can set `buildOptions.drafts: true` in `astro.config.mjs`, or pass the `--drafts` flag when running `astro build`. Markdown pages which do not have the `draft` property set are not affected. 
+`draft: true` is an optional front matter value that will mark an individual `.md` page or post as "unpublished." By default, this page will be excluded from the site build.
```

Comment:
```suggestion
`draft: true` is an optional frontmatter value that will mark an individual `.md` page or post as "unpublished." This will result in the page being excluded from the final build of the site.
```

Reply:
The reason this says "by default" is because there *is* a configuration setting wherein you can specify that you want draft posts built anyway. But the default is to not build them, and you have to opt-in to build them.  (frontmatter is indeed, one word, though! I'll do a full document check for that!)